features/step_definitions/smokey_steps.rb

```diff
@@ -228,10 +228,10 @@ def should_see(text)
 Then /^I should be at a location path of "(.*)"$/ do |location_path|
   if @response
     uri = URI(@response['location'])
-    expect(uri.path).to eq(location_path)
+    expect(normalise_path(uri.path)).to eq(location_path)
```

Comment:
I would just write `uri.path.chomp('/')` inline - it's easier to read and unambiguous as to the intent.

Reply:
Thanks - I've used `chomp` below, though `"/".chomp` would result in "/" which isn't 100% what we want (unless we normalise both the expected and observed path) so I've kept in the helper method.

Reply:
Ohh sorry, I didn't twig that you want to remove only one trailing `/`. I think that makes the behaviour of the method a bit unintuitive though: `foo/` -> `foo` but `foo//` -> `foo/` - that's at least a little bit odd.

Reply:
Or in other words, the method kinda needs unit tests 🙈

Reply:
Sorry ignore me, GitHub wasn't showing the updated code hence my confusion. It's all good.